### PR TITLE
feat(blog): added initial blog entry for 2024 Aug

### PR DESCRIPTION
**Pull request by krisantrobus**
Added a new blog post for latest Paste newsletter.

### CODE REVIEW

File: packages/paste-website/src/components/ResponsiveBorderedImage.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ const ResponsiveBorderedImage: React.FC<ImageProps> = (props) => {
       borderRadius="borderRadius20"
       overflow="hidden"
     >
-      <Image placeholder="blur" style={{ height: "100%", maxWidth: "100%" }} {...props} />
+      <Image placeholder="blur" style={{ height: "100%", minWidth: "100%" }} {...props} />
```
Comment (krisantrobus) [inline packages/paste-website/src/components/ResponsiveBorderedImage.tsx:18]: Seeing an issue where corner ornament would't fill width. Observed no breaking changes.

File: packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+- Added new [icons](/components/icon) for RCSCapable and SupportRequest.
+
+#### Updates
+
+- [Badge](/components/badge#counter) has a new notification_counter variant.
+
+And thank you for all these contributions, Joe Fehrman!:
+
+- [Form Pill Groups](/components/form-pill-group#large) now have a large variant to support the Filter pattern
+- [User Dialog](/components/user-dialog) has a smaller default size based on feedback from our consumers
```
Comment (PixeledCode) [inline packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx:124]: these are missing period at the end.

File: packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+Bravo to Andrés and his attention to detail!
+
+Sign up for [Comms Platform UX office hours](https://airtable.com/appszlqNhBPkOhjqA/shrRkluqJRg1fade0/tblbiQQjRGR9tXygz) if you're interested in design advice and feedback for your own work, too.
+
+## 👀 What we're working on next
+
+| Feature                        | Description                                                                                                                                                         |
+| ------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Timeline component**         | The Timeline component represents a sequence of events or steps in a process.                                                                                       |
```
Comment (serifluous) [inline packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx:167]:
```suggestion
| **Timeline component**         | A Timeline is a visual representation of events displayed in chronological order.                                                                                       |
```

Updating this line based on @roxanagomez 's spec

File: packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx (hunk 2 above)
Comment (krisantrobus) [reply packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx:125]: Great catch, they were missing periods in the [spec](https://docs.google.com/document/d/1DrrGpV0BZ6A5UEaCGzSeBgF2o_yTpyTFhupNFz_2LVM/edit) too so I left them, off. Not sure grammatically which is correct so leaving off for now

Comment (serifluous) [reply packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx:125]: whoops, @PixeledCode is right here. Full sentences in lists should have periods: https://paste.twilio.design/foundations/content/product-style-guide#lists

Good catch!

File: packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx [hunk 4]
```diff
@@ -0,0 +1,189 @@
+export const meta = {
+  title: "Paste Design System Newsletter - August 2024 Edition",
+  slug: "/blog/2024-08-23-paste-newsletter/",
+  date: "2024-08-23",
+  author: "Kristian Antrobus",
+  avatar: "https://avatars.githubusercontent.com/u/55083528",
+  excerpt: "What's new in Paste in July 2024? Corner Ornament, updated filters, and more!",
```
Comment (nkrantz) [inline packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx:7]:
```suggestion
  excerpt: "What's new in Paste in August 2024? Corner Ornament, updated filters, and more!",
```

File: packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx [hunk 5]
```diff
@@ -0,0 +1,189 @@
+export const meta = {
+  title: "Paste Design System Newsletter - August 2024 Edition",
+  slug: "/blog/2024-08-23-paste-newsletter/",
+  date: "2024-08-23",
+  author: "Kristian Antrobus",
```
Comment (nkrantz) [inline packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx:5]: missed this on the last one, we typically have done "Paste" as the author for the newsletters, but totally up to you! just so you know you don't _have_ to put your name up there if you don't want to lol

File: packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+
+We added typography and effect variables to the [Paste Components & Themes](https://www.figma.com/file/l2qtpQVMmXzQPVB3rEAA8i/Paste-Components-%26-Themes?type=design&node-id=3015-46646&mode=design&t=LavBWI8WtuCpWbz0-0) Figma library for more accurate swapping between Twilio, Twilio Dark, Evergreen, Default, and Dark themes.
+
+[Learn more about how to swap modes](https://help.figma.com/hc/en-us/articles/15343816063383-Modes-for-variables#h_01H3ADKDF7JBTRV1Z5P1X1DZVE).
+
+### Additional updates
+
+#### New
+
+- Added new [icons](/components/icon) for RCSCapable and SupportRequest.
```
Comment (nkrantz) [inline packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx:116]:
```suggestion
- Added new [icons](/components/icon) for RCSCapable and SupportRequest
```
just want to either have or not have periods for all these bullets

File: packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx (hunk 5 above)
Comment (krisantrobus) [reply packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx]: Na I'll keep it as Paste haha. If anyones name deserves to be there is Sarahs 🤣

**Review by nkrantz — APPROVED** [review]
🚀

**Review by serifluous — APPROVED** [review]

**Pull request merged**